db/migrate/20250425161749_remove_billing_information_and_record_source_id_from_ite_client_registries.rb

```
@@ -0,0 +1,7 @@
+class RemoveBillingInformationAndRecordSourceIdFromIteClientRegistries < ActiveRecord::Migration[7.0]
```

RuboCop (1.75.5)

[convention] Missing top-level documentation comment for `class RemoveBillingInformationAndRecordSourceIdFromIteClientRegistries`.

(Style/Documentation)